### PR TITLE
Finish document for job error handler #1581

### LINKED ISSUE #1581: Finish document for job error handler
Please add more document for job error handler both English and Chinese:

The documents are in https://github.com/apache/shardingsphere/tree/master/docs/document/content/user-manual/elasticjob-lite/configuration/built-in-strategy/sharding.en.md and https://github.com/apache/shardingsphere/tree/master/docs/document/content/user-manual/elasticjob-lite/configuration/built-in-strategy/sharding.cn.md

Please add: 

- [x] Fill config for job error handler send message via Email
- [x] Fill config for job error handler send message via Dingtalk
- [x] Fill config for job error handler send message via Wechat

### PR DESCRIPTION
**Pull request by wwj-go**
Fixes #1581.

Changes proposed in this pull request:
- Finish document for job error handler

### CODE REVIEW

File: docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.en.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ Maven POM:
...
-| Name                      | Description | Default Value     |
-| ------------------------- |:----------- |:----------------- |
-| webhook                   | TODO        | None              |
-| connectTimeoutMillisecond | TODO        | 3000 milliseconds |
-| readTimeoutMillisecond    | TODO        | 5000 milliseconds |
+| Name                             | Description                                                                | Default Value     |
+| -------------------------------- |:---------------------------------------------------------------------------|:----------------- |
+| wechat.webhook                   | The webhook address of the wechat robot                                    | None              |
+| wechat.connectTimeoutMillisecond | The timeout period for establishing a connection with the wechat server    | 3000 milliseconds |
+| wechat.readTimeoutMillisecond    | The timeout period for reading available resources from the wechat server  | 5000 milliseconds |
```
Comment (TeslaCN) [inline docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.en.md]: How to understand "available resources"? Is that HTTP response?

Comment (wwj-go) [reply docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.en.md]: That is a HTTP response

Comment (wwj-go) [reply docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.en.md]: For wechat and dingTalk，we send the error message to a URL named webhook by HTTP POST method

File: docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.cn.md [hunk 2]
```diff
@@ -47,19 +47,18 @@ Maven 坐标：
...
-| email.cc       | TODO        | 无                       |
-| email.bcc      | TODO        | 无                       |
-| email.debug    | TODO        | false                    |
+| 属性名    | 说明                 | 默认值                   |
+| -------- |:-------------------- |:------------------------ |
+| host     | 邮件服务主机地址      | 无                       |
+| port     | 邮件服务主机端口      | 无                       |
+| username | 用户名               | 无                       |
+| password | 密码                 | 无                       |
+| useSsl   | 是否启用ssl加密传输   | 无                       |
```
Comment (terrymanu) [inline docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.cn.md]: There is default value here

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.cn.md (hunk 2 above)
Comment (wwj-go) [reply docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.cn.md]: I will correct this  and other same case.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**